src/Splitter.sol

```diff
@@ -115,6 +121,7 @@ contract Splitter {
     }
 
     function cage(uint256) external auth {
```

Comment:
Wondering if we should still `cage` the `flap` in case in the future is necessary to do some settlement.

Reply:
Can you give an example of what you mean by settlement?
Is it for a future case where the flapper also hold funds?

Reply:
Yeah as discussed, this is thinking on a future Flapper that might hold funds. Anyway it seems we don't think is necessary to do it, so feel free to solve this one.